crates/bevy_ecs/src/query/iter.rs

```diff
@@ -35,7 +35,7 @@ where
     /// This does not check for mutable query correctness. To be safe, make sure mutable queries
     /// have unique access to the components they query.
     /// This does not validate that `world.id()` matches `query_state.world_id`. Calling this on a `world`
-    /// with a mismatched WorldId is unsound.
+    /// with a mismatched [`WorldId`] is unsound.
```

Comment:
```suggestion
    /// with a mismatched [`WorldId`][crate::world::WorldId] is unsound.
```

(This is a private item)